packages/pwa-buildpack/lib/WebpackTools/configureWebpack/getModuleRules.js

```diff
@@ -56,7 +56,7 @@ getModuleRules.js = async ({
     const astLoaders = [
         {
             // Use custom loader to enable warning reporting from Babel plugins
-            loader: 'buildbus-babel-loader',
+            loader: 'babel-loader',
```

Comment:
This fixes Storybook but likely isn't the solution we want because now `buildbus-babel-loader.js` sits in the repository unused.

I wasn't sure how or if we wanted to actually "install" `buildbus-babel-loader` anywhere - the other loaders are called directly by the new targetables code: https://github.com/magento/pwa-studio/blob/develop/packages/pwa-buildpack/lib/WebpackTools/targetables/TargetableESModule.js#L81.

Reply:
I do not know enough about buildbus to feel comfortable about accepting this scope in this PR. Please back this out and we'll bring Storybook back in scope of [PWA-1074](https://jira.corp.magento.com/browse/PWA-1074).

Reply:
Good call. Removed in https://github.com/magento/pwa-studio/pull/2793/commits/b3212ad8892a3456993e105cc93673d7951415f8.